project/profiles/capacity4more/themes/c4m/kapablo/theme/preprocess.inc

```diff
@@ -84,6 +84,10 @@ function kapablo_preprocess_page(&$variables) {
   // Add search form.
   $variables['search_form'] = drupal_get_form('c4m_search_form');
 
+  if (og_context()) {
```

Comment:
@amitaibu if I remove this `if` I get an error:
![error___capacity4more_and_preprocess_inc_-_capacity4more_-___applications_mamp_htdocs_capacity4more_](https://cloud.githubusercontent.com/assets/3472902/16261598/dcfa2258-3874-11e6-8683-f682e7d511b9.png)